packages/server-reason-react-ppx/cram/client-component-e2e.t/run.t

```diff
@@ -91,7 +91,7 @@
                         module J = {
                           [@ocaml.warning "-unboxable-type-in-prim-decl"]
                           [@mel.internal.ffi
-                            "����\000\000\000\011\000\000\000\005\000\000\000\r\000\000\000\012���A�$name@"
+                            "����\000\000\000\011\000\000\000\005\000\000\000\r\000\000\000\012���A�$name@"
```

Comment:
@davesnx There is something strange here. I made no changes to this file besides the one at line 20, as you can see on the diff below.

![Captura de Tela 2025-05-05 às 19 42 57](https://github.com/user-attachments/assets/ae3019f3-33cf-4f67-91e5-7f4fc5b7f165)

Do you know what is happening?
I tested and saw that when I changed the name of the tool from extract-client-components to `extract-client-modules`, it started to happen.

Reply:
You can see the error here as well: https://github.com/ml-in-barcelona/server-reason-react/actions/runs/14847718294/job/41685229888

As you can see in the image, the test has a strange fail:
<img width="944" alt="Captura de Tela 2025-05-05 às 23 28 56" src="https://github.com/user-attachments/assets/2970a8b4-b904-42e6-8e61-b3df091c2570" />